src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -151,6 +185,70 @@ const CreateCacheWizard = (props) => {
         history.push('/');
     };
 
+    const onGoToStep = (id, prevId) => {
+        // console.log(id, prevId);
+    };
+
+    const getNextStep = (event, activeStep, callback) => {
+        // debugger;
+        console.log('getNextStep', event, activeStep);
+
+        event.stopPropagation();
+        if (activeStep.name === t('caches.create.getting-started.nav-title')) {
+            if (formCreateType === "configure") {
+                setStateObj(
+                    {
+                        ...stateObj,
+                        showEditCodeStep: false,
+                        showConfigurationStep: true,
+                        showReviewStep: true
+                    },
+                    () => callback()
+                );
+            } else {
+                setStateObj(
+                    {
+                        ...stateObj,
+                        showEditCodeStep: true,
+                        showConfigurationStep: false,
+                        showReviewStep: false
+                    },
+                    () => callback()
+                );
+            }
+        }
+        else if (activeStep.name === t('caches.create.edit-config.nav-title')) {
```

Comment:
I don't think is a good idea to use the translator here. steps should have an id that is not depending of the name. and then use the translator only to display